source/base/timer.cc

```diff
@@ -179,24 +242,11 @@ double Timer::cpu_time() const
 {
   if (running)
     {
-#if defined(DEAL_II_HAVE_SYS_TIME_H) && defined(DEAL_II_HAVE_SYS_RESOURCE_H)
-      rusage usage;
-      getrusage (RUSAGE_SELF, &usage);
-      const double dtime =  usage.ru_utime.tv_sec + 1.e-6 * usage.ru_utime.tv_usec;
-      const double running_time = dtime - current_lap_starting_cpu_time + accumulated_cpu_time;
-
-      // in case of MPI, need to get the time passed by summing the time over
```

Comment:
Since this is always done I think a better place to describe what is going on is the documentation (which I will update).